packages/core/src/input/pointer/Pointer.ts

```diff
@@ -0,0 +1,33 @@
+import { Vector2 } from "@oasis-engine/math";
+
+export enum PointerPhase {
```

Comment:
Write this in a seperate file in `input/enums/` folder.

Reply:
Done.